apps/website/cli/renderWebsite.ts

```diff
@@ -20,6 +20,10 @@ export default {
 
         // 2. Get exports from `site` stack, for `args.env` environment.
         const apiOutput = getStackOutput({ folder: "api-ddb", env: args.env });
+        if (!apiOutput) {
```

Comment:
This little think blows up with errors during every deploy. @Pavel910 says it's some old stuff and should be removed entirely. I leave it for now, because I don't know the project that well to clean it up.

Reply:
Folder name `api-ddb` should be just `api`, and that's it.

Reply:
This should be `api` as @Pavel910 suggested.